src/config/test_postake_bootstrap.mlh

```diff
@@ -16,5 +16,6 @@
 
 [%%define integration_tests true]
 [%%define force_updates false]
+[%%define fixup_localhost_ips_for_testing true]
```

Comment:
I don't see this variable being used in our codebase. Can you remove it?

Reply:
This is just for testing convenience, it lets us run test locally without doing `ifconfig` stuff.